logstash-core/lib/logstash/api/modules/root.rb

```diff
@@ -19,10 +19,35 @@ module LogStash
   module Api
     module Modules
       class Root < ::LogStash::Api::Modules::Base
+
+        HEALTH_STATUS = [
+          Java::OrgLogstashHealth::Status::GREEN.to_s,
+          Java::OrgLogstashHealth::Status::YELLOW.to_s,
+          Java::OrgLogstashHealth::Status::RED.to_s
```

Comment:
@yaauie maybe you have an idea of how to reuse the Status enum somehow, maybe there's a better way than this.